src/components/landingPage/LandingPage.tsx

```diff
@@ -1,10 +1,20 @@
 import React from "react";
+import "./LandingPage.scss";
+import { Link } from "react-router-dom";
 
 export const LandingPage: React.FunctionComponent = () => {
   return (
     <main>
-      <h1>Landing Page</h1>
-      <p>Landing page</p>
+      <img src="/images/image-of-rovers-on-mars.jpg" alt="images of rovers on Mars" className="headline-image"/>
```

Comment:
Prefer space before closing tag:
```suggestion
      <img src="/images/image-of-rovers-on-mars.jpg" alt="images of rovers on Mars" className="headline-image" />
```

Reply:
noted - thanks